src/activities/ActivityUsageEntity.js

```diff
@@ -387,7 +387,15 @@ export class ActivityUsageEntity extends Entity {
 		if (!dates) return;
 		if (!this._hasDatesChanged(dates.startDate, dates.dueDate, dates.endDate, dates.startDateType, dates.endDateType)) return;
 
-		const datesActionAndFields = this._generateDatesAction(dates.startDate, dates.dueDate, dates.endDate, true, dates.startDateType, dates.endDateType);
+		const datesActionAndFields = this._generateDatesAction(
+			dates.startDate,
+			dates.dueDate,
+			dates.endDate,
+			true, //validateOnly
+			dates.startDateType,
+			dates.endDateType,
+			dates.displayInCalendar
+		);
```

Comment:
Nit: Should we update the docstrings while we're here? [L384](https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/514/files#diff-b81c79c0e332ef29b7d3ef8ff523e2fe24ba38fa929791c2538a2a76c484a404R384) and [L406](https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/514/files#diff-b81c79c0e332ef29b7d3ef8ff523e2fe24ba38fa929791c2538a2a76c484a404R406)